settings.gradle

```diff
@@ -142,6 +142,10 @@ include(':envoy')
 
 include(':grpc-proxy')
 
+if (includeJavadoc == 'true') {
```

Comment:
this probably isn't going to work like you think it will?  I didn't think settings could read properties.  Try cleaning your project and trying again.

Reply:
I *think* it works...